src/server/engine_shard_set.cc

```diff
@@ -30,6 +30,18 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+// memory defragmented related flags
+ABSL_FLAG(float, mem_defrag_threshold,
+          1,  // The default now is to disable the task from running! change this to 0.05!!
```

Comment:
can you please change dragonfly_test invocation in ci.yml to run with `mem_defrag_threshold=0.05` so we can see if this triggers the bug?

Reply:
add a new entry to the CI.yaml file fo this